Tests failing on my machine, ubuntu amd64

Hi, I run the tests from https://github.com/nats-io/nats-streaming-server/commit/d884137af1d03aa1ea3d553ccdf20819e1c7a5f6 and I got following error:

```
=== RUN   TestSignalTrapsSIGTERM
--- FAIL: TestSignalTrapsSIGTERM (2.01s)
    server_test.go:188: process not started yet
        1 - /mnt/ssd/repositories/nats-streaming-server/server/server_test.go:187
        2 - /mnt/ssd/repositories/nats-streaming-server/server/server_test.go:301
        3 - /mnt/ssd/repositories/nats-streaming-server/server/signal_test.go:140
        4 - /usr/local/go/src/testing/testing.go:909
        5 - /usr/local/go/src/runtime/asm_amd64.s:1357
=== RUN   TestTimeoutReader
```

I run it as documented [here](https://docs.nats.io/nats-streaming-server/install#installing-from-the-source)

```
go test -v -p=1 ./... -sql=false
```

I tried it multiple without success

--- a/server/signal_test.go
+++ b/server/signal_test.go
@@ -131,7 +131,7 @@ func (sc *stderrCatcher) Write(p []byte) (n int, err error) {
 }
 
 func TestSignalTrapsSIGTERM(t *testing.T) {
-	// This test requires that the
+	// This test requires that the server be installed.
 	cmd := exec.Command("nats-streaming-server")
 	sc := &stderrCatcher{}
 	cmd.Stderr = sc
@@ -144,7 +144,7 @@ func TestSignalTrapsSIGTERM(t *testing.T) {
 		if ready {
 			return nil
 		}
-		return fmt.Errorf("process not started yet")
+		return fmt.Errorf("process not started yet, make sure you `go install` first!")
 	})
 	syscall.Kill(cmd.Process.Pid, syscall.SIGTERM)
 	cmd.Wait()

